godel/config/godel.yml

```diff
@@ -1,12 +1,15 @@
 plugins:
   resolvers:
-    - https://palantir.bintray.com/releases/{{GroupPath}}/{{Product}}/{{Version}}/{{Product}}-{{Version}}-{{OS}}-{{Arch}}.tgz
+    - "https://palantir.bintray.com/releases/{{GroupPath}}/{{Product}}/{{Version}}/{{Product}}-{{Version}}-{{OS}}-{{Arch}}.tgz"
   plugins:
     - locator:
-        id: "com.palantir.godel-dep-plugin:dep-plugin:1.1.0"
+        id: com.palantir.godel-mod-plugin:mod-plugin:1.0.1
         checksums:
-          darwin-amd64: 946b0def510a7e94b46bb635a67f00c05243505e2a2ef9b0b7dab45b0437be1f
-          linux-amd64: cc2729e7f25c0121f841e237238c11514ee4c1376c458a1c4997ee86de17b209
+          darwin-amd64: df22922bacfe4e4e7c255607a0aace176205f04ae001f3746276fcfab1780e01
+          linux-amd64: a2697b3d504bb37c2fd8831a66c7014927a6d94e4dfb9765b4764354370a1ab6
+environment:
+  GO111MODULE: "on"
+  GOFLAGS: "-mod=vendor"
```

Comment:
Let's leave this flag out since we don't commit a `vendor` directory for these library projects.